app/api/uploadFile/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const imageTypes = ["image/jpeg", "image/jpg", "image/png"];
+
+export const generateKeyName = (fileType: string) => {
```

Comment:
The name function is not so explicit.

Maybe make explicit it's about file extension like `generateFileKeyName`.

Reply:
resolved!